lib/theme_store/git_importer.rb

```diff
@@ -122,6 +122,7 @@ def clone!
     when "ssh"
       clone_ssh!
     else
+      return clone_folder! if @url == "tmp/component"
```

Comment:
The PR description says `/tmp/component`, but this checks for a different string.